test/components/popup.spec.tsx

```diff
@@ -0,0 +1,269 @@
+import React = require('react');
+import ReactDOM = require('react-dom');
+import {selectDom} from 'test-drive';
+import { ClientRenderer, expect, waitFor } from 'test-drive-react';
+import {PopupDemo} from '../../demo/components/popup-demo';
+import {HorizontalPosition, Popup, VerticalPosition} from '../../src/components/';
+
+const popup = 'POPUP';
+const container = 'POPUP_DEMO_DIV';
+
+describe('<Popup />', function() {
+    const clientRenderer = new ClientRenderer();
+    const bodySelect = selectDom(document.body);
+    const testContainer = document.body.appendChild(document.createElement('div'));
```

Comment:
if this is your test anchor, name it as such.
why is this here and not inside the test?